canonical/lxd-ui
# Add operations

## Done

- added an operations page
- added artificial creation rows on top of the instance table

Fixes #291 

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described in the [Readme](https://github.com/canonical/lxd-ui#setting-up-for-development).
2. Perform the following QA steps:
    - check the operations page
    - check the instance table after launching a new image
